imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -0,0 +1,50 @@
+// +build linux
+
+package imageboot
+
+import (
+	"flag"
+	"os"
+	"syscall"
+	"testing"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *runtest {
+		os.Exit(m.Run())
+	} else {
+		os.Exit(0)
+	}
+}
+
+func TestGuestBoot(t *testing.T) {
+	err := syscall.Uname(&syscall.Utsname{})
+
+	if err != nil {
+		t.Fatalf("couldn't get system information, image boot failed")
+	}
+
+}
+
+func TestGuestReboot(t *testing.T) {
+	_, err := os.Stat("/tmp/boot")
+	if os.IsNotExist(err) {
+		// first boot
+		if _, err := os.Create("/tmp/boot"); err != nil {
```

Comment:
no need to use /tmp, we are the sole owners of the VM. and /tmp is ephemeral on some systems. recommend either / or /root directories for marker file creation

Reply:
change to use `/`